_src/Evasions/techniques/os-features.md

```diff
@@ -204,12 +208,68 @@ bool Cuckoo::CheckUnbalancedStack() const {
 <p></p>
 Signature recommendations are not provided as it's pretty tricky to track such a behavior on malware side.
 
+<br />
+
+<h3><a class="a-dummy" name="detect-wine">3. Detect Wine</a></h3>
+
+<hr class="space">
+
+The <tt>MulDiv</tt> [API](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-muldiv) is being called with specific arguments (<tt>`MulDiv(1, 0x80000000, 0x80000000)`</tt>) which should logically return 1 - however, due to a bug with the ancient implementation on Windows, it returns 2.
+
+There are more known evasion methods to detect Wine like the good old check of searching for the existence of one of Wine’s exclusive APIs such as <tt>`kernel32.dll!wine_get_unix_file_name`</tt> or <tt>`ntdll.dll!wine_get_host_version`</tt>).
```

Comment:
Can you add 'wine_get_host_version' here:
https://evasions.checkpoint.com/src/Evasions/techniques/processes.html#check-if-specific-functions-are-present-in-specific-libraries? And then provide a link